test/blackbox-tests/mel-unwrap-for-obj.t

```diff
@@ -5,14 +5,12 @@ Test `@mel.unwrap` in `@mel.obj`
   > external func :
   >   param:string ->
   >   polyParam:([ \`Str of string | \`Int of int ][@mel.unwrap]) ->
-  >   unit ->
-  >   _ Js.t = ""
```

Comment:
This looks like a regression. Shouldn't `mel.obj` return a `'a Js.t` value?

Reply:
It's still a Js.t, but it gets around the weak variable warning